kubernetes/cmsweb/scripts/deploy-srv.sh

```diff
@@ -2,7 +2,7 @@
 # helper script to deploy given service with given tag to k8s infrastructure
 
 if [ $# -lt 2 ]; then
-     echo "The required parameters for service and tag are missing. Please use deploy-srv.sh <service> <tag> <env> "
```

Comment:
@amaltaro  Yes, this can cause a mixup of parameters 3 and 4. I think rather than removing the env variable entirely, we can make the script take keyword arguments because when you're doing multiple deployments at the same time, it helps to make sure that you are deploying in the correct environment.